resources/qml/pages/ContactsPage.qml

```diff
@@ -77,22 +77,22 @@ Page {
                     }
                     Label {
                         id: jidId;
-                        text: jid;
+                        text: qsTr("jid");
```

Comment:
This is an id which gets replaced by c++ code with the contact jabber id. Not translateable.